content/news/XXXX-XX-XX-dvs-internals-pt1.md

```diff
@@ -0,0 +1,156 @@
+authors: Jan Holthuis
+tags: 2.3, timecode, dvs, vinyl control, serato, cd
+status: draft
+# Analog/Digital diagram via matplotlib: https://gist.github.com/Holzhaus/46ab8ea32d3b89025a0a640db171a14b
+
+
+Since the dawn of DJing, spinning vinyl records has never really gone out of fashion.
+Even when CDs became popular in 90s and most music listeners happily phased out their record players, the classical "two turntables and a mixer" setup stayed the epitome of DJing.
+In recent years, digital DJ controllers became more popular, but there are still many DJs that stick with their Technics SL-1210s turntables.
+
+There's a reason for that: the tactile feeling of manipulating music with your hands is something that controllers have had trouble archieving.
+Motorized controllers like the Rane One try to provide a similar workflow, but it's still to be seen if they will have a notable impact on the prevalence of vinyl.
+
+On the other hand, going digital has its benefits:
+Jumping to different positions instantly inside a track using hotcues - without picking up, moving and dropping the needle - just isn't possible with vinyl records, and carrying around heavy crates of records isn't the best you can do for your back.
+
+Digital Vinyl Systems (DVS) aim to provide the best of both worlds and combine the flexibility and power of digital DJing with the tactile control of traditional vinyl DJing.
+But how does it actually work?
+
+## The Basics
+
+Instead of a traditional record that contains music, a "control" record is used.
+Such a record contains a special signal that is processed by the computer.
+It transforms the audio data into information about direction ("Is the record playing forwards or backwards?"), pitch ("How fast is the record spinning?") and position information ("At which position of the record is the needle?") and uses that track to manipulate the playing track in software.
+
+There are many formats out there:
+[VirtualDJ](http://www.virtualdj.com/buy/controlvinyl.html), [Rekordbox](https://www.pioneerdj.com/product/features/software/rekordbox-dvs-control-vinyl/), [Numark](https://www.numark.com/product/virtualvinyl), [MixVibes](https://www.mixvibes.com/cross-dj-4/), [Traktor](https://www.native-instruments.com/en/products/traktor/digital-vinyl/traktor-scratch-a10/) and [Serato](https://serato.com/dj/pro/expansions/dvs?dvs=dvs-ready) all provide their own solution.
+Mixxx (and the underlying [xwax](https://xwax.org/) library) [support the latter 3 formats](https://manual.mixxx.org/2.3/en/chapters/vinyl_control.html#supported-timecode-media).
+
+Serato is one of the most popular and robust digital vinyl systems.
+There's also Serato Control CD which basically works the same, just for CDJs instead of turntables.
+Since the latter essentially works the same as the vinyl version and can be [downloaded for free from the Serato Website](https://serato.com/controlcd/downloads), I'm going to explain how it works based on that format.
```

Comment:
The "works" sounds kind of double with the previous sentence. How about: 

```suggestion
Since it contains essentially the same control track and it can be [downloaded for free from the Serato Website](https://serato.com/controlcd/downloads), I'm going to explain how it works based on that format.
```

Reply:
It doesn't contain the same control track though, the LFSR uses different taps. I rephrased it.